plugins/file-open-save/core/src/main/java/org/pentaho/repo/dialog/RepositoryOpenSaveDialog.java

```diff
@@ -94,6 +98,19 @@ public void open( String directory, String state, String filter, String origin )
     }
   }
 
+  private static String getClientPath(){
+    Properties properties = new Properties();
+    try {
+      InputStream inputStream =
+        FileHelper.class.getClassLoader().getResourceAsStream("project.properties");
```

Comment:
What's this FileHelper? I'd say use the current class to get the classloader from.